lms/djangoapps/teams/views.py

```diff
@@ -877,6 +877,14 @@ def post(self, request):
         except User.DoesNotExist:
             return Response(status=status.HTTP_404_NOT_FOUND)
 
+        course_module = modulestore().get_course(team.course_id)
+        if ('max_team_size' in course_module.teams_configuration and
+            team.users.count() >= course_module.teams_configuration['max_team_size']):
```

Comment:
As long as `paver run_quality` says it's okay, then it's okay. I believe that should be an okay style for conditionals.